### PR TITLE
Consume LibG from NuGet package

### PR DESCRIPTION
**Pull request by aparajit-pratap**
### Purpose

JIRA: https://jira.autodesk.com/browse/DYN-966
Update LibG reference to NuGet package for LibG - This PR goes hand-in-hand with https://git.autodesk.com/Dynamo/LibG/pull/699.

WIP: This PR will be merged once LibG packages for all branches have been posted on NuGet.

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@mjkkirschner @QilongTang 

### FYIs

@smangarole

### CODE REVIEW

File: test/Libraries/WorkflowTests/packages.config [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="DynamoVisualProgramming.LibG_225" version="2.1.0" targetFramework="net47" />
```
Comment (mjkkirschner) [inline test/Libraries/WorkflowTests/packages.config]: I'd imagine there will be 3 of these references?

File: src/DynamoCore/DynamoCore.csproj [hunk 2]
```diff
@@ -346,7 +345,6 @@ limitations under the License.
     <ProjectReference Include="..\NodeServices\DynamoServices.csproj">
       <Project>{ef879a10-041d-4c68-83e7-3192685f1bae}</Project>
       <Name>DynamoServices</Name>
-      <Private>False</Private>
```
Comment (QilongTang) [inline src/DynamoCore/DynamoCore.csproj]: Is this intentional, I suspect VS silently removed it?

File: test/Libraries/WorkflowTests/packages.config (hunk 1 above)
Comment (aparajit-pratap) [reply test/Libraries/WorkflowTests/packages.config]: Yes, still generating those ...

File: src/DynamoCore/DynamoCore.csproj (hunk 2 above)
Comment (aparajit-pratap) [reply src/DynamoCore/DynamoCore.csproj]: Yes, it's unintentional.

File: src/DynamoCore/DynamoCore.csproj [hunk 3]
```diff
@@ -365,14 +365,14 @@ limitations under the License.
...
+      <LibGInterfaceDll Include="$(SolutionDir)\packages\DynamoVisualProgramming.LibG_225_0_0.2.1.0\ProtoGeometry\LibG.Interface.dll" />
+      <ProtoGeometryXml Include="$(SolutionDir)\packages\DynamoVisualProgramming.LibG_225_0_0.2.1.0\lib\ProtoGeometry\ProtoGeometry.XML" />
       <ExternProtoGeometryUICulture Include="$(SolutionDir)..\extern\ProtoGeometry\$(UICulture)\*" />
       <ExternProtoGeometryLibGLocale Include="$(SolutionDir)..\extern\ProtoGeometry\libg_locale\**\*.*" />
-      <ExternLibG223 Include="$(SolutionDir)..\extern\LibG_223_0_1\*" />
-      <ExternLibG224 Include="$(SolutionDir)..\extern\LibG_224_0_1\*" />
-      <ExternLibG2244 Include="$(SolutionDir)..\extern\LibG_224_4_0\*" />
-      <ExternLibG225 Include="$(SolutionDir)..\extern\LibG_225_0_0\*" />
-      <ExternLibG Include="$(SolutionDir)..\extern\LibG_*\*" />
+      <LibG223 Include="$(SolutionDir)\packages\DynamoVisualProgramming.LibG_223_0_1.2.1.0\LibG_223_0_1\*" />
```
Comment (aparajit-pratap) [inline src/DynamoCore/DynamoCore.csproj]: I worry that each time we bump up the version numbers of LibG builds, these package names would update as well and these entries would have to be manually updated over here. I'll try and see if there's a way we can parameterize this.

**Pull request closed without merging**